scripts/extras/foglamp.sudoers

```diff
@@ -1,2 +1,3 @@
-%sudo ALL=(ALL) NOPASSWD: /usr/bin/apt-get -y update, /usr/bin/apt-get -y install foglamp
-
+%sudo ALL=(ALL) NOPASSWD: /usr/bin/apt-get -y update
+%sudo ALL=(ALL) NOPASSWD: /usr/bin/apt-get -y install foglamp
+%sudo ALL=(ALL) NOPASSWD: /bin/tar -C / -xzf *
```

Comment:
If we use the set uid script we should not need this and it will be more secure.

Reply:
Correct. This entry now stands removed. Please see comment above.